recipes/gklib/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set name = "gklib" %}
+{% set version = "5.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/KarypisLab/GKlib/archive/METIS-v{{ version }}-DistDGL-0.5.tar.gz
+  sha256: 52aa0d383d42360f4faa0ae9537ba2ca348eeab4db5f2dfd6343192d0ff4b833
+
+build:
+  number: 0
+  # skip: true  # [win]
```

Comment:
If shared libraries are used instead `run_exports` should probably be used here. If not it might need a metapackage for tracking the ABI.
```suggestion
```

(I can expand on either of these depending on the outcome of my first comment)

Reply:
@chrisburr I added `run_exports` for the shared unix builds in 1342d88 - is that the correct syntax and what you were suggesting?